scipy/optimize/tests/test_quadratic_assignment.py

```diff
@@ -226,7 +219,7 @@ def test_specific_input_validation(self):
             )
 
         K = np.random.rand(3, 3)
-        K = _doubly_stochastic(K, tol=1)
+        K = _doubly_stochastic(K, tol=1.5)
```

Comment:
This was initially much small. I made it 1. It needs to be even higher to trigger the error? Do you know why the  double stochastic matrix isn't detecting matrices generated with a lower tolerance?

Reply:
The reason the test was failing was because the matrix generated by _doubly_stochastic() with `tol=1` was actually doubly stochastic, so it was correctly not causing an error. I think it would save trouble, since there is randomness in generating `K` this way, if we just manually set K for this test, especially since its only 3x3. If you agree I'll make that change